### PR TITLE
nrf54h20: Fix EXMIF build failures

### PR DESCRIPTION
**Pull request by krish2718**
Build failure started with this https://github.com/zephyrproject-rtos/zephyr/commit/fa86c518f6704c9d782cf2bbce4808cc6ee01a77

### CODE REVIEW

**Review by JordanYates — DISMISSED** [review]

**Comment by JordanYates** [comment]
Pulled both of these revert commits into #73302, which fixes both issues with the sample in one PR, letting CI pass.

**Review by e-rk — APPROVED** [review]

**Review by anangl — DISMISSED** [review]

**Comment by FRASTM** [comment]
Seems taht stm32 label added by the  [zephyrbot] is not relevant, is it

File: boards/st/stm32h745i_disco/stm32h745i_disco_stm32h745xx_m7.dts [hunk 1]
```diff
@@ -46,6 +46,7 @@
 		led0 = &green_led;
 		pwm-led0 = &green_pwm_led;
 		sw0 = &user_button;
+		spi-flash0 = &mt25ql512ab1;
```
Comment (FRASTM) [inline boards/st/stm32h745i_disco/stm32h745i_disco_stm32h745xx_m7.dts:49]: this part of the PR # https://github.com/zephyrproject-rtos/zephyr/pull/73027/

**Comment by krish2718** [comment]
Closing PR as all fixes are part of other PRs linked above.

**Pull request closed without merging**